install/jakartaee/bin/sig-test.map

```diff
@@ -155,7 +155,7 @@ javax.enterprise.deploy=1.2_se6
 jakarta.security.jacc=1.2_se6
 
 # JAXB 
-javax.xml.bind=2.1_se6
+jakarta.xml.bind=3.0_se6
```

Comment:
This is old sig map file. We use install/jakartaee/bin/sig-test_se8.map for jdk8 runs where we do not have sig test for jakarta.xml.bind mapping or the sig file in this repo. Will need to check whether we need to add jakarta.xml.bind along with platform run. jakarta.xml.bind standalone sig test is running separately though.

Reply:
I'll revert this file change, now that I understand that this file is for historical reference (I assume, due to standard edition 6 reference).

Reply:
> Copyright year may need to be updated from "Copyright (c) 2018" to "Copyright (c) 2018, 2020"

@RohitKumarJain thanks, I reverted the docker/jaxrtck.sh file change which was the only one missing a copyright date change.  We don't really need to update docker/jaxrtck.sh as we are removing JAXR.